frontend/src/components/buttons/ButtonEdit.vue

```diff
@@ -1,20 +1,25 @@
 <template>
-  <v-btn class="px-3 px-sm-4" color="primary" v-bind="$attrs" v-on="$listeners">
-    <v-icon :left="$vuetify.breakpoint.smAndUp" size="150%">{{ icon }}</v-icon>
-    <span class="d-none d-sm-block">
-      <slot>{{ $tc('global.button.edit') }}</slot>
-    </span>
-    <span class="d-sr-only d-sm-none">
-      <slot>{{ $tc('global.button.edit') }}</slot>
-    </span>
-  </v-btn>
+  <IconButton
```

Comment:
Nice to use the generic IconButton. Should we do this also for Edit/Delete/Add?

Reply:
Probably. But I rather look at the usages generally. I want to establish a concept when to use which button style, in a way that serves the primary use case and is not too distracting for secondary actions.